backend/src/gpt_providers/xai_gpt.py

```
@@ -9,13 +9,18 @@ def __init__(self, api_key: str = None):
         if api_key is None:
             api_key = os.getenv('XAI_API_KEY')
             if not api_key:
-                raise ValueError("API key not found. Please set the XAI_API_KEY environment variable.")
+                raise ValueError(
+                    "API key not found. Please set the XAI_API_KEY "
+                    "environment variable."
+                )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)